.github/workflows/deb_release.yml

```diff
@@ -0,0 +1,73 @@
+name: Debian packages release
+
+on:
+  workflow_dispatch:
+    inputs:
+      config:
+        description: JSON of options
+        type: string
+        default: '{}'
+        required: true
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Save the config options
+        run: |
+          echo '${{ inputs.config }}' | jq > config.json
+      - name: Archive the release configuration
+        uses: actions/upload-artifact@v3
+        with:
+          name: config.json
+          path: config.json
+      - name: Install dependencies
+        run: |
+          sudo apt update -qq
+          sudo apt install -qq -y bumpversion devscripts python3-launchpadlib
+          git clone https://git.launchpad.net/~checkbox-dev/checkbox/+git/support
+      - name: Checkout checkbox monorepo
+        uses: actions/checkout@v3
+        with:
+          repository: canonical/checkbox
+          fetch-depth: 0
+          path: src/checkbox
+      - name: Check for new commits
+        run: |
+          support/release/git/release_deb_monorepo.py clone --config config.json
+      - name: Bump versions
+        run: |
+          git config --global user.email "sylvain.pineau@canonical.com"
+          git config --global user.name "Sylvain Pineau"
+          support/release/git/release_deb_monorepo.py bump --part major
+      - name: Create changelog
+        run: |
+          support/release/git/release_deb_monorepo.py changelog
+      - name: Archive changelog
+        uses: actions/upload-artifact@v3
+        with:
+          name: Changelog
+          path: changelog
+      - name: Push release tags
+        run: |
+          support/release/git/release_deb_monorepo.py push
+      - name: Update the PPA recipe and kick-off the builds
+        env:
+          LP_CREDS: credentials
```

Comment:
this is a _path_ to credentials, not actual credentials, meaning we have multiple problems with it:
* automation doesn't consider this to be secret, so the safety precautions may not work (like redacting stuff on the fly)
* this is a relative path (as in ./credentials) that's super fragile
* the rm invocation later on is not guaranted, so this credential file should be made more ephemeral (like used from secrets directly)

Reply:
Fixed